docs/sdk/client-side-sdks/flutter.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Flutter SDK
+sidebar_position: 6
+---
+
+# DevCycle Flutter Client SDK
+
+The Flutter Client SDK for DevCycle! This SDK uses our Client SDK APIs to perform all user segmentation
+and bucketing for the SDK, providing fast response times using our globally distributed edge workers
+all around the world.
+
+The SDK is available as a package on Pub. It is also open source and can be viewed on GitHub.
+
+[![Pub](https://img.shields.io/pub/v/devcycle_flutter_client_sdk)](https://img.shields.io/pub/v/devcycle_flutter_client_sdk)
+[![GitHub](https://img.shields.io/github/stars/devcyclehq/flutter-client-sdk.svg?style=social&label=Star&maxAge=2592000)](https://github.com/devcyclehq/flutter-client-sdk)
+
+## Requirements
+
+This version of the DevCycle Flutter Client SDK supports a minimum of Flutter 2, iOS 12 and Android API Version 21.
+
+## Installation
+
+### Pub
+
+The SDK can be installed into your Flutter project by adding the following to your `pubspec.yaml`:
+
+```dart
+devcycle_flutter_client_sdk: ^1.0.0
```

Comment:
do we include the version in the docs for our other sdks? this seems like it's likely to get out of date as we release new versions — can we reference the version in github instead?

Reply:
I'll take a look to see if we can do a generic install command 👀